src/content/api/loaders.md

```diff
@@ -643,3 +643,11 @@ module.exports = function(source) {
   return match[0];
 };
 ```
+
+## Logging
+
+Logging API is available since the release of webpack 4.37. When `logging` is enabled in [`stats configuration`](/configuration/stats/#stats) and/or when [`infrastructure logging`](/configuration/other-options/#infrastructurelogging) is enabled, loaders may log messages which will be printed out in the respective logger format (stats, infrastructure).
+
+- Loaders should prefer to use `this.getLogger()` for logging which is a shortcut to `compilation.getLogger()` with loader path and processed file. This kind of logging is stored to the Stats and formatted accordingly. It can be filtered and exported by the user.
```

Comment:
```suggestion
- Loaders should prefer to use `this.getLogger()` for logging which is a shortcut to `compilation.getLogger()` with loader path and processed file. This kind of logging is stored to the Stats and formatted accordingly. It can be filtered and exported by the loader author for the user.
```

User you mean the loader author or the final user of the loader?

Reply:
the final user via webpack config

Reply:
changed to `webpack user` @montogeek